ansible/roles/rhel_zero_touch_user/.yamllint

```diff
@@ -0,0 +1,13 @@
+---
+extends: default
```

Comment:
Please use our standard .yamllint and customize from there as needed.
```
extends: ../../../tests/static/.yamllint
```

Reply:
Changed,